flloat/ldlf.py

```diff
@@ -151,6 +171,8 @@ def to_nnf(self):
         return type(self)(self.r.to_nnf(), self.f.to_nnf())
 
 
+# TODO: this class is not used: propositionals are passed directly to LDLf
+#  operators. Is this fine, we may just correct all type hints in this module.
 class LDLfPropositionalAtom(LDLfFormula):
     """An LDLf propositional formula.
 
```

Comment:
A LDLfFormula to represent propositional atoms would be a cleaner way to proceed. In fact, all operators would join `LDLfFormula` instead of a mix of LDLf and propositionals. However, even before my chages this class was not used at all. So, I opted for the fastest way to make it work. We may later use this class and delete FiniteTraceWrapper.

Its truth function could be similar to that of FiniteTraceWrapper instead of